### DISCUSSION TITLE
Social Login for China (WeChat)

### DISCUSSION THREAD

**Discussion by Aypzie** [supabase/supabase#29711] [Feature Requests]
Would it be possible to have WeChat added as an authentication provider for Social Login? Unfortunately the majority of services currently supported by Supabase are not available in China, so adding WeChat OAuth (with their 1.3 billion monthly active users) would quickly alleviate this problem.

**Comment by ssudaraka** [comment]
Hi @Aypzie

At the moment, Supabase has no plans to introduce support for WeChat auth. However, there are plans to introduce support for implementing custom OAuth providers in the future, which should allow you to implement WeChat OAuth with Supabase. Unfortunately, I do not have a timeline for when this will be introduced.

**Reply by lisirrx** [reply]
> Hi @Aypzie
> 
> At the moment, Supabase has no plans to introduce support for WeChat auth. However, there are plans to introduce support for implementing custom OAuth providers in the future, which should allow you to implement WeChat OAuth with Supabase. Unfortunately, I do not have a timeline for when this will be introduced.

Any update on this? I believe the customized login method will be very helpful.
